index.html

```diff
@@ -69,5 +69,130 @@ <h2 class="connect">LET'S CONNECT</h2>
+              </div>               
+            </div>
+          </div>
+        </div>
+        <div class="cardWork br">
+          <img id="snapshot_sec">
+          <div class="left Block_sec">
+            <div class="primary text_sec">
+              <h class="project title_sec">Multi-Post <br>
+                Stories</h>
+              <div >
+                <ul class="frame">
+                  <li class="client_tag_list"><h class="client_canopy">CANOPY</h></li>
+                  <span class="counter"></span>
+                  <li class="client_tag_role"><h class="client_role_text">Back End Dev</h></li>
+                  <span class="counter_tw"></span>
+                  <li class="client_tag_year"><h class="client_yr_text">2015</h></li>
+                </ul>
+              </div>
+            </div>
+            <div class="client_txt_bd">
+              <p4>A daily selection of privately personalized reads; no accounts or sign-ups required.</p4>
+            </div>
+            <ul class="tag_list">
+              <li class="html_tag"><h class="html_txt">html</h></li>
+              <li class="css_tag"><h class="css_txt">css</h></li>
+              <li class="java_tag"><h class="java_txt">javaScript</h></li>
+            </ul>
+            <div id="action">
+              <div class="butt">
+                <button class="btn_text" href="#">See Project</button>
+              </div>               
+            </div>
+          </div>
+        </div>
+        <div class="cardWork th">
+          <img id="snapshot_th">
+          <div class="left block_th">
+            <div class="primary text_th">
+              <h3 class=" project title_th">Tonic</h3>
+              <div >
+                <ul class="frame">
+                  <li class="client_tag_list"><h class="client_canopy">CANOPY</h></li>
+                  <span class="counter"></span>
+                  <li class="client_tag_role"><h class="client_role_text">Back End Dev</h></li>
+                  <span class="counter_tw"></span>
+                  <li class="client_tag_year"><h class="client_yr_text">2015</h></li>
+                </ul>
+              </div>
+            </div>
+            <div class="client_txt_bd">
+              <p4>A daily selection of privately personalized reads; no accounts or sign-ups required.</p4>
+            </div>
+            <ul class="tag_list">
+              <li class="html_tag"><h class="html_txt">html</h></li>
+              <li class="css_tag"><h class="css_txt">css</h></li>
+              <li class="java_tag"><h class="java_txt">javaScript</h></li>
+            </ul>
+            <div id="action">
+              <div class="btnn">
+                <button class="btn_text" href="#">See Project</button>
+              </div>               
+            </div>
+            </div>
+          </div>
+        </div>
+        <div class="cardWork fr">
+          <img id="snapshot_fr">
```

Comment:
Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image, also you need to put the attribute SRC and put inside the URL of the image.